src/pybind/mgr/dashboard/settings.py

```diff
@@ -38,14 +38,23 @@ def has_default_value(name):
 class SettingsMeta(type):
     def __getattr__(cls, attr):
         default, stype = getattr(Options, attr)
-        return stype(mgr.get_config(attr, default))
+        if stype == bool and str(mgr.get_config(attr,
```

Comment:
What happens if the string value is ``true``?

Reply:
The `else` clause on line 45 will be evaluated. The conversion of `"true"` to bool (`bool('true')`) will return `True`.

Reply:
Maybe you should add a comment to the else block that the ``"true"`` value will be handled here, too. This is a in-depth knowledge of Python which is surely not known by everyone.